diode-server/proto/reconciler/v1/reconciler.proto

```diff
@@ -36,8 +37,67 @@ message RetrieveIngestionDataSourcesResponse {
   repeated IngestionDataSource ingestion_data_sources = 1;
 }
 
+// ChangeSetError represents an error when applying a change set
+message ChangeSetError {
+  message Details {
```

Comment:
thoughts about nested structures vs global? I've added as nested because it does not make that sense being used separetely

Reply:
Yes, it should be fine 👍